core/src/controllers/rows.js

```diff
@@ -66,15 +75,23 @@ const listTableRows = async (req, res) => {
   // will filter by name like '%John%' and age like '%20%'
 
   const filters = _filters.split(',').map((filter) => {
-    const [field, value] = filter.split(':');
-    return { field, value };
+    let [key, value] = filter.split(':');
```

Comment:
I'd like this to be more pessimistic, please @AbegaM .  I'd like us to handle when a filter doesn't match our expected list, and throw a suitable error message.

Reply:
@IanMayo Ah, I haven't considered that. I will modify the code to throw errors.

Reply:
@thevahidal, I have added error handling to the feature request. Please review it and let me know if there is anything I can modify.